lib/os/fdtable.c

```diff
@@ -332,9 +329,8 @@ ssize_t write(int fd, const void *buf, size_t sz)
 
 	return res;
 }
-FUNC_ALIAS(write, _write, ssize_t);
 
-int close(int fd)
+int zvfs_close(int fd)
```

Comment:
are we going to group these `zvfs_*` functions into a file?

Reply:
That would be part of the ZVFS task - in this PR, I'm mainly concerned with correcting years of erroneous Kconfig options.